services/QuillLMS/engines/evidence/app/services/evidence/vertex_ai/text_classifier.rb

```diff
@@ -7,6 +7,9 @@ class TextClassifier < ApplicationService
       DISPLAY_NAMES = 'displayNames'
       PREDICT_API_TIMEOUT = 5.0
 
+      PREDICTION_EXCEPTION_CLASSES = [Google::Cloud::InternalError, Google::Cloud::UnknownError]
+      PREDICTION_NUM_RETRIES = 2
```

Comment:
Let's change this to `1` for now. This is a synchronous call that the student is waiting on. Given we allow each API call to take up to 5 seconds, I think waiting 15 secs for 3 attempts is a little too long, especially given there are additional APIs to hit by other rules.